src/current/v24.3/wal-failover.md

```diff
@@ -0,0 +1,470 @@
+
+No: If the side disk for WAL failover was **not** pre-configured before the cluster was started, changing the value of the [`storage.wal_failover.unhealthy_op_threshold`]({% link {{ page.version.version }}/cluster-settings.md %}#setting-storage-wal-failover-unhealthy-op-threshold) cluster setting to something less than the [`storage.max_sync_duration`]({% link {{ page.version.version }}/cluster-settings.md %}#setting-storage-max-sync-duration) cluster setting **will not have any effect and when the disk stalls, WAL failover will not trigger**. In other words, you cannot enable WAL failover at runtime for a single store cluster if you did not [pre-configure a side disk](#single-store-config).
+
+The recommended default value of the [`storage.wal_failover.unhealthy_op_threshold`]({% link {{ page.version.version }}/cluster-settings.md %}#setting-storage-wal-failover-unhealthy-op-threshold) cluster setting is `100ms`.
+
+### 9. Can I change the log buffering on a node before enabling WAL failover?
+
+Yes.
+
+### 10. Can I change the value of the `storage.max_sync_duration` cluster setting at cluster runtime?
+
+Yes. To change [`storage.max_sync_duration`]({% link {{ page.version.version }}/cluster-settings.md %}#setting-storage-max-sync-duration), issue a statement like the following:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+SET CLUSTER SETTING storage.max_sync_duration = '${desired_duration}';
+~~~
+
+### 11. When will Cockroach revert to writing a store's WAL on it's primary storage device? What if the secondary storage device stalls?
+
+CockroachDB will monitor the latencies of the primary storage device in the background. As soon as latencies return to acceptable levels, the store will begin writing to the primary device. If the secondary stalls while in use by WAL failover, WAL failover will be unable to limit tail latencies, and the user workload will experience latencies until either the primary or the secondary recovers.
+
+### 12. If there are more than 2 stores, will the WAL failover cascade from store A to B to C?
+
+Store _A_ will failover to store _B_, store _B_ will failover to store _C_, and store _C_ will failover to store _A_, but store A will never failover to store C.
+
+However, the WAL failback operation will not cascade back until **all drives are available** - that is, if store _A_'s disk unstalls while store _B_ is still stalled, store _C_ will not failback to store _A_ until _B_ also becomes available again. In other words, _C_ must failback to _B_, which must then failback to _A_.
```

Comment:
Are you using failback here to specifically refer to failover in the other direction? If so, apologies for suggesting it be changed in other places. I assumed they were synonymous and that we needed to stick to one term.

```suggestion
However, the WAL failover operation will not cascade back until **all drives are available** - that is, if store _A_'s disk unstalls while store _B_ is still stalled, store _C_ will not fail over to store _A_ until _B_ also becomes available again. In other words, _C_ must fail over to _B_, which must then fail over to _A_.
```